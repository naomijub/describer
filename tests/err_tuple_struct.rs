use describer::Describe;
 
#[derive(Describe)]
struct MyStruct(Option<bool>, String);
 
fn main() {
}