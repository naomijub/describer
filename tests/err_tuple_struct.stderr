error: The Tuple Struct variant is not yet supported
 --> tests/err_tuple_struct.rs:4:16
  |
4 | struct MyStruct(Option<bool>, String);
  |                ^^^^^^^^^^^^^^^^^^^^^^
